backend/dataall/base/config.py

```diff
@@ -1,18 +1,36 @@
 """Reads and encapsulates the configuration provided in config.json"""
 
-import json
 import copy
-from typing import Any, Dict
+import json
+import logging
 import os
 from pathlib import Path
+from typing import Any, Dict
+
+from dataall.base.utils import Parameter
+
+ENVNAME = os.getenv('envname', 'local')
+
+log = logging.getLogger(__name__)
 
 
 class _Config:
-    """A container of properties in the configuration file
-    and any other that can be specified/overwritten later in the application"""
+    """
+    A container of properties that configure data.all.
+    * Reads from config.json
+    * Optionally updates/overwrite from SSM /dataall/{ENVNAME}/configjson if it exists
+    * Updates dynamically at runtime via set_property
+    """
 
     def __init__(self):
+        log.info('reading data.all config from local file')
         self._config = _Config._read_config_file()
+        try:
+            if ssm_config := _Config._read_config_parameter():
```

Comment:
Where does `ssm_config` come from? I can't see the declaration. Same about `configjson` (line 79).

Reply:
There is double work happening here: first, try to read file and then ssm_config. Can we somehow do only one option? Or is it unnecessary and it's quick enough not to worry about double loading?

Reply:
My thought here is that we use the SSM parameter to overwrite the config.json so we need to read both.
The reason for overwriting instead of providing the entire thing is for brevity and for keeping compatibility with upstream changes.